### ISSUE TITLE
Support for Simdjson

### ISSUE THREAD

**Issue by danilobarion1986** [intridea/multi_json#198]
Hi,
Is it possible to add support for [simdjson](https://github.com/saka1/simdjson_ruby) gem?

**Comment by rwz** [comment]
TLDR: It's possible, but I don't think it's going to happen.

This gem is in maintenance mode and no new features and versions are being developed at the moment. One of the goals is to keep this gem backwards compatible with older versions of Ruby (until 1.8) to support legacy codebases depending on it. I personally do not recommend using this gem in newer projects.

Also, MultiJSON's interface is intended to provide convenience over performance, and simdjson seems to be very performance oriented. If performance is what you're after, you have another reason to not use MultiJSON and do a more custom integration instead.

Furthermore, it doesn't seem like the suggested gem is mature and established enough to justify building an adapter. Pre 1.0 versions tend to change their public APIs quite often and introduce a lot of maintenance burden.

You're certainly welcome to fork MultiJSON and build your own adapter and use the fork, but (again), if you want performance, I recommend not using MultiJSON.

**Comment by danilobarion1986** [comment]
Thanks for your reply, that really makes sense.
